ui/admin/tests/e2e/tests/credential-store-static.spec.js

```diff
@@ -146,3 +146,211 @@ test('Static Credential Store (User & Key Pair)', async ({ page }) => {
+  const orgs = JSON.parse(execSync('boundary scopes list -format json'));
+  const org = orgs.items.filter((obj) => obj.name == orgName)[0];
+  const projects = JSON.parse(
+    execSync(`boundary scopes list -format json -scope-id ${org.id}`)
+  );
+  const project = projects.items.filter((obj) => obj.name == projectName)[0];
+  const targets = JSON.parse(
+    execSync(`boundary targets list -format json -scope-id ${project.id}`)
+  );
+  const target = targets.items.filter((obj) => obj.name == targetName)[0];
+
+  const session = JSON.parse(
+    execSync(`boundary targets authorize-session -id ${target.id} -format json`)
+  );
+  const retrievedUser = session.item.credentials[0].credential.username;
+  const retrievedPassword = session.item.credentials[0].credential.password;
+
+  if (process.env.E2E_SSH_USER != retrievedUser) {
+    throw new Error(
+      'Stored User does not match. EXPECTED: ' +
+        process.env.E2E_SSH_USER +
+        ', ACTUAL: ' +
+        retrievedUser
+    );
+  }
+  if (testPassword != retrievedPassword) {
+    throw new Error('Stored Password does not match');
+  }
+});
+
+test('Static Credential Store (JSON)', async ({ page }) => {
+  const orgName = await createNewOrg(page);
+  const projectName = await createNewProject(page);
+  await createNewHostCatalog(page);
+  const hostSetName = await createNewHostSet(page);
+  await createNewHostInHostSet(page);
+  const targetName = await createNewTarget(page);
+  await addHostSourceToTarget(page, hostSetName);
+
+  const credentialStoreName = 'Credential Store ' + nanoid();
+  await page
+    .getByRole('navigation', { name: 'Resources' })
+    .getByRole('link', { name: 'Credential Stores' })
+    .click();
+  await page.getByRole('link', { name: 'New' }).click();
+  await page.getByLabel('Name', { exact: true }).fill(credentialStoreName);
+  await page.getByLabel('Description').fill('This is an automated test');
+  await page.getByRole('group', { name: 'Type' }).getByLabel('Static').click();
+  await page.getByRole('button', { name: 'Save' }).click();
+  await expect(
+    page.getByRole('alert').getByText('Success', { exact: true })
+  ).toBeVisible();
+  await page.getByRole('button', { name: 'Dismiss' }).click();
+  await expect(
+    page.getByRole('link', { name: credentialStoreName })
+  ).toBeVisible();
+
+  const credentialName = 'Credential ' + nanoid();
+  await page.getByRole('link', { name: 'Credentials', exact: true }).click();
+  await page.getByRole('link', { name: 'New', exact: true }).click();
+  await page.getByLabel('Name', { exact: true }).fill(credentialName);
+  await page.getByLabel('Description').fill('This is an automated test');
+  await page.getByRole('group', { name: 'Type' }).getByLabel('JSON').click();
+  await page.getByText('{}').click();
+  const testName = 'name-json';
+  const testPassword = 'password-json';
+  await page.keyboard.type(
+    `"username": "${testName}", "password": "${testPassword}"`
```

Comment:
Maybe we add one more property in the JSON to differentiate it from the username/password test?